### PR TITLE
feat(component): new builder paging control component

### PR DESCRIPTION
**Pull request by sf-tejas-nadkarni**

### CODE REVIEW

**Comment by ashmc** [comment]
@tnadkarni2 can you also update the README table to include this new component please, and you are missing adding your custom labels to the [labels folder](https://github.com/forcedotcom/commerce-on-lightning-components/tree/release/force-app/main/default/labels)

File: force-app/main/default/lwc/builderSearchPagingControl/builderSearchPagingControl.js [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+        const nextPageNumber = Number(this.currentPageNumber) + 1;
+        this.dispatchUpdateCurrentPageEvent(nextPageNumber);
+    }
+
+    /**
+     * Handler for the 'click' event from the page number button.
+     * @param {Event} event The event object
+     */
+    handlePaginationPage(event) {
+        let pageNumber = parseInt(event.target.value, 10);
```
Comment (seckardt) [inline force-app/main/default/lwc/builderSearchPagingControl/builderSearchPagingControl.js:173]: Should be `const`

Comment (seckardt) [reply force-app/main/default/lwc/builderSearchPagingControl/builderSearchPagingControl.js:173]: There are a number of warnings flagged by the GH actions that should also be heeded.

**Comment by seckardt** [comment]
@tnadkarni2 Have you also verified that pushing the custom code to an org as well as using the component in Experience Builder works?

**Comment by sf-tejas-nadkarni** [comment]
@seckardt @ashmc I have created a new PR for the builder component https://github.com/forcedotcom/commerce-on-lightning-components/pull/22
Tested the component in the experience bundle and its working as expected. I will close this PR

**Pull request closed without merging**